tests/test_update_manager_coverage_97.py

```
@@ -104,3 +105,167 @@ def test_database_update_manager_initialization_handles_dependency_errors(
+        assert hash(wrapper1) == hash(wrapper1_copy)  # Same hash for equal objects
+        assert hash(wrapper1) != hash(wrapper2)  # Different hash for different objects
+
+        # Test that wrappers with same path have same hash (important for set/dict usage)
+        path_set = {wrapper1, wrapper1_copy}
+        assert len(path_set) == 1  # Should be deduplicated
+
+    def test_calculate_checksum_with_cache_data(self) -> None:
+        """Test checksum calculation with cache data (lines 497-498)."""
+        from core.food_apis.update_manager import DatabaseUpdateManager
+
+        manager = DatabaseUpdateManager(cache_dir="/tmp")
```

Ruff (0.13.3)

Probable insecure usage of temporary file or directory: "/tmp"

(S108)